build/style.css

```diff
@@ -95,6 +95,11 @@ h1 {
   color: #ed9121;
 }
 
+.filled-box.show-perceived-truth {
+  font-family: "Comic Sans", "Comic Sans MS", "Apple Chancery", "Bradley Hand", "papyrus", serif;
```

Comment:
Could you drop the Comic Sans fonts for perceived-truth? I like the way it bounces between comic-sans for lies and the fancier fonts for truths, leaving neutral at helvetica